include/odp/api/spec/schedule_types.h

```diff
@@ -400,6 +428,29 @@ typedef struct odp_cache_stash_prio_config_t {
  * Schedule group parameters
  */
 typedef struct odp_schedule_group_param_t {
+	/** Group specific priority configuration */
+	struct {
+		/** Maximum number of priorities to be supported for this group
+		 *
+		 *  Further constrains number of priorities to be supported at
+		 *  group level. The priority span should not exceed what was
+		 *  configured globally (see odp_schedule_config_t::num_prios)
+		 *  and queues with priorities outside this span should not be
+		 *  configured under the group.
+		 *
+		 *  Use 0 to not limit priorities at group level. 0 by default.
+		 */
+		uint32_t num;
+
+		/** Starting priority for this group
+		 *
+		 *  Starting and lowest priority in the priority range for this
+		 *  group, entire group specific range spanning from 'start' to
+		 *  'start' + 'num' - 1.
+		 */
+		odp_schedule_prio_t start;
```

Comment:
Perhaps `min` could be clearer name, or not...

Reply:
Spec talks about ranges and spans so 'start' is probably OK (i.e. start of a span/range ...).